command/operator_diagnose.go

```diff
@@ -502,6 +502,11 @@ SEALFAIL:
 			}
 			if ln.Config.TLSDisableClientCerts {
 				diagnose.Warn(listenerTLSContext, "TLS for a listener is turned on without requiring client certs.")
+
+			}
+			err = diagnose.TLSMutualExclusionCertCheck(ln.Config)
+			if err != nil {
+				diagnose.Warn(listenerTLSContext, "TLSDisableClientCerts and TLSRequireAndVerifyClientCert should not both be set.")
```

Comment:
You might want to use `TLSDisableClientCerts and TLSRequireAndVerifyClientCert should not both be set.` as a prefix and include the error you throw in this Warning.